controller/src/beerocks/master/wfa_ca.cpp

```diff
@@ -8,8 +16,91 @@
 
 #include <easylogging++.h>
 
+#include <algorithm>
+
 using namespace beerocks;
 
+std::string wfa_ca::wfa_ca_status(eWfaCaStatus status)
+{
+    switch (status) {
+    case RUNNING:
+        return "RUNNING";
+    case INVALID:
+        return "INVALID";
+    case ERROR:
+        return "ERROR";
+    case COMPLETE:
+        return "COMPLETE";
+    default:
+        break;
```

Comment:
There should be some error logging here.

Also, (nitpick), I generally prefer to use the same pattern for all switch cases, so return here as well instead of break (and no return at end of function).

Reply:
also, function name should fit what it does - something like `wfa_ca_status_to_string()`